src/class_singleWorker.py

```diff
@@ -299,7 +299,7 @@ def doPOWForMyV2Pubkey(self, adressHash):
             BMConfigParser().set(
                 myAddress, 'lastpubkeysendtime', str(int(time.time())))
             BMConfigParser().save()
-        except:
+        except:  # noqa:E722
```

Comment:
`configparser.NoSectionError`, can pass

also add a general exception handler and log warning

Reply:
Updated